batch-nft-reveal/app/components/collection/RevealInfo.tsx

```diff
@@ -0,0 +1,32 @@
+import { Box, Container, Text, Heading } from '@chakra-ui/react'
+import { useNextRevealAmount } from '../../hooks/useNextRevealAmount'
+import { useNextRevealInterval } from '../../hooks/useNextRevealInterval'
+import { AddressProp } from '../../types/AddressProp'
+
+export const RevealInfo = (props: AddressProp): JSX.Element => {
+  const { contractAddress } = props
+
+  const nextRevealBatchSize = useNextRevealAmount(contractAddress)
+  const nextRevealInterval = useNextRevealInterval(contractAddress)
+
+  return (
+    <Container
+      mt={5}
+      maxWidth="100%"
+      border="1px solid #000"
+      padding="5"
+      textAlign="center"
+    >
+      <Heading>Next reveal after</Heading>
```

Comment:
Title should change when reveal is pending (ex: 0 NFTs or 0 minutes), then after the reveal counters must reset and appear again.